### PR TITLE
Update dig.md

### PR DESCRIPTION
**Pull request by zhangtreefish**
offer the option that I reply to the bird after digging out

### CODE REVIEW

**Comment by zhangtreefish** [comment]
Please review this, @casey-collab

File: english/dig/dig.md [hunk 1]
```diff
@@ -18,4 +18,6 @@ Do you:
 
 [Try to scare the bird?](bird/bird.md)
 
-[Take the marshmallow out of your pocket and offer it to the bird?](marshmallow_to_bird/marshmallow_to_bird.md)
\ No newline at end of file
+[Take the marshmallow out of your pocket and offer it to the bird?](marshmallow_to_bird/marshmallow_to_bird.md)
+
+[Reply the greeting by saying "How do you do?" to the bird.](reply-greeting/reply-greeting.md)
```
Comment (casey-collab) [inline english/dig/dig.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**